docs/proposals/proposals_process_proposal.MD

```diff
@@ -0,0 +1,107 @@
+# Proposal: Proposals - have a process and documentation
+
+*Author*: Robyn Bergeron <@robynbergeron>
+
+*Date*: 04/03/2016
+
+- Status: New
+- Proposal type: community development process
+- Targeted Release: Forever, until we improve it more at a later date.
+- PR for Comments: https://github.com/ansible/ansible/pull/14802#
+- Estimated time to implement: 2 weeks at most
+
+Comments on this proposal prior to acceptance are accepted in the comments section of the pull request linked above.
+
+## Motivation
+Define light process for how proposals are created and accepted, and document the process permanently in community.html somewhere.
+
+The following suggested process was created with the following ideas in mind:
+- Transparency: notifications, decisions made in public meetings, etc. helps people to know what is going on.
+- Avoid proliferation of multiple comments in multiple places; keep everything in the PR.
+- Action is being taken: Knowing when and where decisions are made, and knowing who is the final authority, gives people the sense that things are moving.
+- Ensure that new features or enhancements are added to the roadmap and release notes.
+
+### Problems
+Proposals are confusing. Should I write one? Where do I put it? Why can’t I find any documentation about this? Who approves things? This is why we should have a light and unbureaucratic process.
+
+## Solution proposal
+This proposal has multiple parts:
+- Proposed process for submitting / accepting proposals
+- Suggested proposal template
+
+Once the process and template are approved, a PR will be submitted for documenting the process permanently in documentation, as well as a PR to ansible/docs/proposals for the proposal template.
+
+### Proposed Process
+1: PROPOSAL CREATION
+- Person making the proposal creates the proposal document in ansible/docs/proposals via PR, following the proposal template.
+- Author of proposal PR updates the proposal with the PR # / link.
+- Notify the community that this proposal exists.
```

Comment:
...via email to ansible-devel, I presume? Are there any other accepted mechanisms? (I would say not.)